examples/Makefile

```diff
@@ -12,6 +12,8 @@ ifneq ($(USE_RTTI), 1)
 	CXXFLAGS += -fno-rtti
 endif
 
+CFLAGS += -Wstrict-prototypes
+
```

Comment:
Should a comparable flag also be added for CMAKE?

Reply:
I've added this flag to Makefile as well.  This should prevent future changes that break strict prototypes

Reply:
Also added to CMakeLists.txt and verified passing with cmake -Wdev